snap/snapcraft.yaml

```diff
@@ -29,8 +29,39 @@ apps:
       - network-bind
 
 parts:
+
+  lensfun:
+    plugin: cmake
+    source: https://github.com/lensfun/lensfun.git
+    source-commit: v0.3.2
+    configflags:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/usr
+      - -DBINARY_PACKAGE_BUILD=On
+
+    build-packages:
+      - python3
+      - libglib2.0-dev
+      - libpng-dev
+      - doxygen
```

Comment:
Can we disable documentation generation so we don't need to pull this for building?

Reply:
I'll take a look - I just pulled the list from the debian package `Build-Depends` ;-)